impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -82,21 +108,22 @@ std::shared_ptr<Texture> MakeDownsampleSubpass(
         // creates a halo effect. This compensates for when the expanded clip
         // region can't give us the full gutter we want.
         Vector2 texture_size = Vector2(input_texture->GetSize());
-        Quad vertices =
+        Quad guttered_uvs =
```

Comment:
This math may assume that the centroid of the uv's is 0.5, 0.5 too so it may not work if that isn't the case? @bdero